content/articles/dnsimple-services.markdown

```diff
@@ -1,6 +1,6 @@
 ---
 title: DNSimple Services
-excerpt: DNSimple provides several services that every system connected to the internet needs—such as domain registration, hosted DNS, and SSL certificates.
+excerpt: DNSimple provides several services that every system connected to the internet needs—such as hosted DNS, domain registration, a powerful automation API, One Click DNS Services, and SSL certificates.
```

Comment:
@ARabbitt the excerpt needs to stay within 150 chars limit (due to search engine restrictions). We should pick the best items we want to display.

Reply:
I can massage the excerpt down to 150 Characters.

DNSimple provides services every system connected to the internet needs-such as hosted DNS, domain registration, automation API, and SSL certificates.

Reply:
@weppos I checked on latest thoughts on this (through the scientific method of running a google search) and it looks like the recommended maximum length now is 300 characters. See https://www.google.com/search?q=search+engine+description+length for reference.

Reply:
That's good to know. 300 is defiantly more descriptive than 150.

Reply:
I'm fine to give a try with 300.